### PR TITLE
homepage with placeholder images and sections

### PR DESCRIPTION
**Pull request by rattanakone**
-Added placeholder navigation
-Added a main image
-Added placeholders login/sign up section, search section, and poem of the day
section
-Chose the font style, color, and size
-Chose the site’s colors
-Added placeholder footer
-Added copyright logo
-Added social media section
-Added about, press release, and terms of use section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,64 +22,145 @@
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
-    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
-      <div class="container">
+    <nav class="navbar navbar-default" id="my-nav">
+      <div class="container-fluid">
         <div class="navbar-header">
-          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-2">
```
Comment (mallorycurtisold) [inline index.html:28]: I played around with this a little and took out this < button > section from lines 28(current)-33 and the code still ran great. Does it fulfill a purpose?

**Comment by mallorycurtisold** [comment]
I think this looks great and could go live immediately!

File: index.html (hunk 1 above)
Comment (rattanakone) [inline index.html:28]: Yes that button has a purpose and it is for mobile view when it collapses. We could take it out if you guys don't want mobile view.

Comment (mallorycurtisold) [inline index.html:28]: Ah I see! No, I like the mobile view.

**Pull request merged**